apply_extra.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+set -e
+
+ar x teams.deb
+tar xf data.tar.xz
```

Comment:
This can be done at once via pipe, like this: `ar p teams.deb data.tar.xz | tar -xJf -`, thus reducing IO and used space usage.

Reply:
Neat.